.repo-metadata-full.json

```diff
@@ -1472,4 +1480,4 @@
         "library_type": "GAPIC_AUTO",
         "api_shortname": "workflows"
     }
-}
+}
```

Comment:
Do we care about the new line here? Is a Json file so I am unsure wether or not we really need it.

Reply:
I think it'd be good to have it... let me add it to the script

Reply:
Done in https://github.com/googleapis/google-cloud-php/pull/7375